packages/@adobe/spectrum-css-temp/components/inputgroup/skin.css

```diff
@@ -217,3 +217,88 @@ governing permissions and limitations under the License.
     }
   }
 }
+@media (forced-colors: active) {
+  .spectrum-InputGroup {
+    --spectrum-dropdown-border-color-error: Highlight;
+    --spectrum-dropdown-border-color-key-focus: Highlight;
+    --spectrum-textfield-border-color-error: Highlight;
+    --spectrum-textfield-border-color-hover: Highlight;
+    --spectrum-textfield-border-color-key-focus: Highlight;
+    --spectrum-textfield-quiet-border-color: Canvas;
+    --spectrum-textfield-quiet-border-color-disabled: Canvas;
+    --spectrum-textfield-quiet-border-color-hover: Highlight;
+    --spectrum-textfield-quiet-border-color-key-focus: Highlight;
+    --spectrum-textfield-text-color-disabled: GrayText;
+      forced-color-adjust: none;
+      .spectrum-InputGroup-input {
+        forced-color-adjust: auto;
+     }
+      .spectrum-FieldButton {
+        forced-color-adjust: auto;
+      }  
+    
+  &.is-disabled:not(.spectrum-InputGroup--invalid):not(.spectrum-InputGroup--quiet) .spectrum-FieldButton {
+    border-color: GrayText;
+    border-width: 1px;
+  }
+  
+}
```

Comment:
Indentation is messed up here. I thought this was closing the media query but it is not.